packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,212 @@
+| ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| text    | `inputmode="text"`, `autocomplete="off"`(can be set to valid options for autocomplete based on the context where the plain text is being use for eg: `name`,`username`,`country`, etc.) | `keyboardType="default"`                                                              |
+| search  | `inputmode="search"`, `autocomplete="off"`(can give option to consumer to configure this with more options which are contextual), `enterkeyhint="search"`                               | `keyboardType="default"`, `returnKeyType="search"`                                    |
+| tel     | `inputmode="tel"`, `autocomplete="tel"`, `enterkeyhint="done"`(can be overriden by consumers)                                                                                           | `keyboardType="phone-pad"`, `returnKeyType="done"`(can be overriden by consumers)     |
+| email   | `inputmode="email"`, `autocomplete="email"`, `enterkeyhint="done"`(can be overriden by consumers)                                                                                       | `keyboardType="email-address"`, `returnKeyType="done"`(can be overriden by consumers) |
+| url     | `inputmode="url"`, `autocomplete="off"`, `enterkeyhint="done"`(can be overriden by consumers)                                                                                           | `keyboardType="url"`, `returnKeyType="done"`(can be overriden by consumers)           |
+| numeric | `type="text"`, `inputmode="decimal"`, `autocomplete="off"`, `enterkeyhint="done"`(can be overriden by consumers)                                                                        | `keyboardType="decimal-pad"`, `returnKeyType="done"`(can be overriden by consumers)   |
+
+### Return Key types for web and Native
+
+| Key Type            | Web                       | Native                                   |
+| ------------------- | ------------------------- | ---------------------------------------- |
+| <kbd>↵</kbd>        | `enterkeyhint="enter"`    | NA                                       |
+| <kbd>go</kbd>       | `enterkeyhint="go"`       | `returnKeyType="go"`                     |
+| <kbd>Done</kbd>     | `enterkeyhint="done"`     | `returnKeyType="done"`                   |
+| <kbd>Next</kbd>     | `enterkeyhint="next"`     | `returnKeyType="next"`                   |
+| <kbd>Previous</kbd> | `enterkeyhint="previous"` | `returnKeyType="previous"`(android only) |
+| <kbd>Search</kbd>   | `enterkeyhint="search"`   | `returnKeyType="search"`                 |
+| <kbd>Send</kbd>     | `enterkeyhint="send"`     | `returnKeyType="send"`                   |
+
+## PasswordInput
+
+**Sample Usage**
+
+```tsx
+<PasswordInput
```

Comment:
How do we handle "Show Password" option here?

![image](https://user-images.githubusercontent.com/30949385/179985074-54bb9f6e-0156-43e4-b196-3e0471244119.png)

Reply:
it'll automatically be placed for all the password input fields, user don't have to decide that, system will take that decision